courses/machine_learning/deepdive2/time_series_prediction/2_feature_engineering.ipynb

```diff
@@ -0,0 +1,2158 @@
+{
```

Comment:
<p>Same as in previous notebook. We need to map objectives to code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/646/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>